src/store/builder/index.ts

```diff
@@ -56,6 +56,7 @@ export interface IBuilderState {
     [OperationKind.BLOCK_RELOCATE]: null,
   },
   draggableForExitsByUuid: object,
+  openBlockEditor: boolean,
```

Comment:
let's stick with standard boolean naming, by adding prefix like: is, should, ...
eg: `isBlockEditorOpen`

Reply:
Done